apps/client/src/app/pages/db/fish/fish.component.html

```diff
@@ -1,269 +1,287 @@
-<ng-container *ngIf="xivapiFish$ | async as xivapiFish">
+<ng-container *ngIf="fishCtx.fishId$ | async">
   <ng-container *ngIf="settings.theme as theme">
-    <app-page-loader [loading]="loading">
-      <ng-container *ngIf="gubalData$ | async as data">
-        <div class="details-container" fxLayout="column">
-          <div class="details" fxLayoutAlign="flex-start center" fxLayout="row wrap" fxLayoutGap="5px">
-            <div class="details-block no-icon">
-              <div class="detail-name">
-                {{'DB.FISH.Min_gathering' | translate}}
-              </div>
-              <div class="detail-value">
-                {{data.minGathering | number:'1.0-1'}}
-              </div>
+    <app-page-loader [loading]="loading$ | async">
+      <div class="details-container" fxLayout="column">
+        <div class="details" fxLayoutAlign="flex-start center" fxLayout="row wrap" fxLayoutGap="5px">
+          <div class="details-block no-icon">
+            <div class="detail-name">
+              {{ 'DB.FISH.Min_gathering' | translate }}
             </div>
-            <div class="details-block no-icon">
-              <div class="detail-name">
-                {{'DB.FISH.Min_size' | translate}}
-              </div>
-              <div class="detail-value">
-                {{(data.minSize / 10) | number:'1.0-1'}} ilm
-              </div>
+            <div class="detail-value">
+              {{ (fishCtx.statisticsByFish$ | async)?.data?.stats.aggregate.min.gathering | number: '1.0-1' }}
```

Comment:
I see this `statisticsByFish$` used a lot, it'd be better to have it aliased inside a wrapping `ng-container` imo, then wrapping each card with its own [nz-spin](https://ng.ant.design/components/spin/en)

Reply:
Definitely! I'll address that as part of the UI cleanup.